components/ILIAS/Authentication/classes/class.ilAuthLoginPageEditorGUI.php

```diff
@@ -325,6 +346,7 @@ protected function saveLoginInfo(): void
                     $this->loginSettings->set($settingKey, $this->form->getInput($settingKey));
                 }
             }
+
```

Comment:
In my opinion, the whole `saveLoginInfo` method can be dropped.

Reply:
done